### PR TITLE
Taskcat vfy multi region

### PR DESCRIPTION
**Pull request by sshajiya**
In this PR below are the files added and modified for the feature Verifying CFT with multi region using Taskcat
        modified Files:
   .github/workflows/pipeline.yml   #Added CFT upload code
   templates/nginx-plus-app-protect-ubnt1804-dev.template.yaml #Changed the AppProtectTargetGroup name as APTG as the stack-name with taskcat is exceeding its length while launching.

New files:
        .taskcat.yml
        delete_stack.py
        utils.py

### CODE REVIEW

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Cloud Formation Template
+
+on:
+  push: {branches: master}
+  pull_request: {branches: main}
```
Comment (464d41) [inline .github/workflows/pipeline.yml:5]: We use master not main.

File: .github/workflows/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Cloud Formation Linter with Latest Version
+        uses: scottbrenner/cfn-lint-action@v2
+      - name: Cloud Formation Version & Verification - Ubuntu.
+        run: |
+          cfn-lint ./templates/nginx-plus-app-protect-ubnt1804-dev.template.yaml --ignore-checks W
+          cfn-lint --version
```
Comment (464d41) [inline .github/workflows/pipeline.yml:18]: This step seems redundant. Separate workflow is going to check the format in advance: https://github.com/f5devcentral/aws-waf-solution-template/pull/13

File: .github/workflows/pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        run: |
+          python -m pip install --user boto3
+          python -m pip install --user requests
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{secrets.AWS_ACCESS_KEY_ID}}
+          aws-secret-access-key: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+          aws-session-token: ${{secrets.AWS_SESSION_TOKEN}}
+          aws-region: ${{secrets.AWS_REGION}}
```
Comment (464d41) [inline .github/workflows/pipeline.yml:34]: This step seems redundant. We pass creds as env variables via github secrets so they are avaliable by default. Please check and remove if true.

File: .github/workflows/pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      - name: Launch the Stack
+        uses: ShahradR/action-taskcat@v1
+        with:
+          commands: test run -n
+          update_cfn_lint: true
+          update_taskcat: true         
+      - name: Delete Stack
+        run:
+          python delete_stack.py
```
Comment (464d41) [inline .github/workflows/pipeline.yml:43]: Taskcat deletes a stack automatically. Doesn't it?

File: delete_stack.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+import boto3
+from utils import *
+
+cfn = boto3.resource('cloudformation')
+stack_name=get_stack_name()
+stack = cfn.Stack(stack_name)
+status=stack.delete()
+print(status)
+
```
Comment (464d41) [inline delete_stack.py:10]: If taskcat can delete stack on its own this file is not required.

File: test1.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+import boto3
+from utils import vfy_cft_link
+nginx_def_index="Welcome to nginx"
+
+vfy_status=vfy_cft_link(nginx_def_index)
+if vfy_status:
+  print("NGINX APP PROTECT BASIC PAGE VERIFICATION IS COMPLETED SUCESSFULLY...")
+else:
+  print("Error: NGINX APP PROTECT BASIC PAGE VERIFICATION IS Failed...")
```
Comment (464d41) [inline test1.py:10]: This workflow only verifies that template is deployable in multiple regions. Seems like this file belongs to separate workflow.

File: utils.py [hunk 7]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```
Comment (464d41) [inline utils.py:1]: This workflow only verifies that template is deployable in multiple regions. Seems like this file belongs to separate workflow.

**Review by 464d41 — CHANGES_REQUESTED** [review]

File: .github/workflows/pipeline.yml (hunk 1 above)
Comment (sshajiya) [reply .github/workflows/pipeline.yml:5]: changed

File: .github/workflows/pipeline.yml (hunk 2 above)
Comment (sshajiya) [reply .github/workflows/pipeline.yml:18]: modularized

Comment (sshajiya) [reply .github/workflows/pipeline.yml:18]: created the separate workflow

File: .github/workflows/pipeline.yml (hunk 3 above)
Comment (sshajiya) [reply .github/workflows/pipeline.yml:34]: without session token cred , unable to connect to aws

File: .github/workflows/pipeline.yml (hunk 4 above)
Comment (sshajiya) [reply .github/workflows/pipeline.yml:43]: As informed in the discussion of creating this file, moving this job into another workflow

File: utils.py (hunk 7 above)
Comment (sshajiya) [reply utils.py:1]: done

File: .github/workflows/release_initial_repo.yml [hunk 8]
```diff
@@ -1,12 +0,0 @@
-name: Release Initial Repo
```
Comment (464d41) [inline .github/workflows/release_initial_repo.yml:1]: Do not delete this file.

**Review by 464d41 — CHANGES_REQUESTED** [review]

**Pull request merged**